obspy/clients/fdsn/header.py

```diff
@@ -46,7 +46,7 @@ class FDSNNoDataException(FDSNException):
     "GFZ": "http://geofon.gfz-potsdam.de",
     "ICGC": "http://ws.icgc.cat",
     "INGV": "http://webservices.ingv.it",
-    "IPGP": "http://eida.ipgp.fr",
+    "IPGP": "http://fdsnws.ipgp.fr",
```

Comment:
@claudiodsf can you please also adapt the doctest in `__init__.py`, thanks

Reply:
Done. I updated the commit and force-pushed.